packages/fhir-converter/Dockerfile

```diff
@@ -10,9 +10,9 @@ RUN npm install --only=production --no-fund --no-optional --no-audit
 ### Prod Image                         ##
 #########################################
 FROM node:18-slim
-RUN  apt-get update && apt install libcurl4-gnutls-dev -y && apt autoremove -y
+# RUN  apt-get update && apt install libcurl4-gnutls-dev -y && apt autoremove -y
 COPY --from=build /app /app
 WORKDIR /app
-RUN ["chmod", "+x", "/app/deploy/webapp.sh"]
+# RUN ["chmod", "+x", "/app/deploy/webapp.sh"]
```

Comment:
this is no longer needed since we cleaned up the converter - leaving in just in case for later

Reply:
Can we add a comment adding context to it? In a few month's time we'll look at this and won't know why it was commented out, if can be removed.

Reply:
added comment